2748. Number of Beautiful Pairs/solution.py

```diff
@@ -0,0 +1,31 @@
+class Solution:
+    # This takes in a list of nums and returns each beautiful pair,
```

Comment:
Better to say list of ints rather than "nums" since that's the name of the list. Also, it returns the number of beautiful pairs. I would do one sentence to state this, and a new sentence to explain what a beautiful pair is.